services/ml/execution_impl_android.cc

```diff
@@ -41,47 +42,46 @@ ExecutionImplAndroid::ExecutionImplAndroid(
         offset, length, memory_->MapAtOffset(length, offset)));
     total_length += length;
   }
+}
 
-  int32_t result = ANeuralNetworksExecution_create(compilation->nn_compilation_,
-                                                   &nn_execution_);
-  DLOG(INFO) << "ANeuralNetworksExecution_create: " << result;
+ExecutionImplAndroid::~ExecutionImplAndroid() {
+  ANeuralNetworksCompilation_free(nn_compilation_);
+  DLOG(INFO) << "ANeuralNetworksCompilation_free";
+  // ANeuralNetworksExecution_free(nn_execution_);
+  // DLOG(INFO) << "ANeuralNetworksExecution_free";
+}
+
+void ExecutionImplAndroid::StartCompute(StartComputeCallback callback) {
+  DLOG(INFO) << "ExecutionImplAndroid::StartCompute";
 
+  ANeuralNetworksExecution* nn_execution;
+  int32_t result =
+      ANeuralNetworksExecution_create(nn_compilation_, &nn_execution);
+  DLOG(INFO) << "ANeuralNetworksExecution_create: " << result;
```

Comment:
The `result ` only be used in log, it's better to check the validity such as 

> if (result !=ANEURALNETWORKS_NO_ERROR ) {
>    LOG(ERROR) << "Failed creating ANeuralNetworksExecution with result code " << result;
>   return; 
> }

And remove L.60

Reply:
> The `result ` only be used in log, it's better to check the validity such as
> 
> > if (result !=ANEURALNETWORKS_NO_ERROR ) {
> > LOG(ERROR) << "Failed creating ANeuralNetworksExecution with result code " << result;
> > return;
> > }
> 
> And remove L.60

I have updated codes as you suggested, and will add the verification for results together in another PR, please help to review, thanks very much! @fujunwei